packages/expo/src/proxies/NativeModules.ts

```diff
@@ -100,3 +104,13 @@ export function disableMissingNativeModuleErrors(moduleNames?: string[] | string
     enabled = false;
   }
 }
+
+/**
+ * Access a native module without throwing an error if it doesn't exist.
+ *
+ * @param moduleName Name of module to access
+ * @returns Corresponding native module object, or null if it doesn't exist
+ */
+export function getNativeModuleIfExists(moduleName: string): any {
+  return originalNativeModules[moduleName];
```

Comment:
Viewing this file (on mobile) it looks like we don't overwrite the NativeModules variable in this file. A benefit of not introducing originalNativeModules is that it becomes possible to call this new method even if createProxyForNativeModules hasn't yet been called.

```suggestion
  return NativeModules[moduleName];
```

Reply:
could you try putting `const originalNativeModules = NativeModules;` in module-level scope (say, after the imports) and seeing if that works? I think what's happening is that `NativeModules` is always lazily accessed on the react-native module object, and introducing a new variable (originalNativeModules) will eagerly save the value of NativeModules.

Reply:
Ahh, that makes sense. Yes, this works -- thanks for the suggestion!